tests/Aspire.Hosting.NodeJs.Tests/AddNodeAppTests.cs

```diff
@@ -27,7 +27,7 @@ public async Task VerifyManifest()
                 "..\\foo\\app.js"
               ],
               "env": {
-                "NODE_ENV": "development",
+                "NODE_ENV": "{{builder.Environment.EnvironmentName.ToLowerInvariant()}}",
```

Comment:
This change is unexpected. I don't think we should be changing this test at all in this PR